env.sample

```diff
@@ -0,0 +1,3 @@
+PGHOST=localhost
+PGUSER=root
+PGPASSWORD=""
```

Comment:
Always useful to have sample `.ENV` files.

Add a newline at the end to keep all the code readers happy.

Reply:
ty